kubernetes/eventhandler.go

```diff
@@ -229,3 +266,9 @@ func (*nodePodUpdater) OnAdd(interface{}) {}
 // OnDelete handles delete events on namespaces. Nothing to do, if pods are deleted from this
 // namespace they will generate their own delete events.
 func (*nodePodUpdater) OnDelete(interface{}) {}
+
+// checkMetadata receives labels or annotations maps and checks their equality. Returns True if equal, False if there is a diffrence
+func checkMetadata(newmetadata, oldmetadata map[string]string) bool {
```

Comment:
I don't really like the name of this function. It doesn't convey the meaning of the return value. I think it should be called something like `isEqual`

Reply:
isEqualMetadata is the new one :)

Reply:
The function is now removed and repalced with inline commands